
public class ExceptionsLab {

	public static void main(String[] args) {
	{
		try
		{
		int[] numbers = {1,3,5,7};		
		AddArray(numbers);
	}
catch (ArrayIndexOutOfBoundsException ar)
		{
	System.out.println("I tried to access a location in an array that did not exist");
		}
}
	}
	private static void AddArray(int[] numbers) 
	{
		int sum = 0;
	
		for (int i = 0; i <=numbers.length; i++)
		{
			sum = sum + numbers[i];
		}
			System.out.println("Sum is: " + sum);
	}
}
